Tests/Microsoft.AppCenter.Test.Windows/Microsoft.AppCenter.Test.Windows.csproj

```diff
@@ -136,6 +146,8 @@
     <None Include="CodeCoverage.runsettings">
       <SubType>Designer</SubType>
     </None>
+    <Fakes Include="Fakes\System.fakes" />
+    <Fakes Include="Fakes\mscorlib.fakes" />
```

Comment:
JFYI this introduces requirement of Visual Studio Enterprise

Reply:
we merged other PRs doing that already though